Cargo.toml

```diff
@@ -16,6 +16,11 @@ documentation = "https://docs.rs/octocrab"
 # and only builds that target
 targets = ["x86_64-unknown-linux-gnu"]
 
+[features]
+default = []
+
+rate-limiting = []
```

Comment:
This doesn't have to be done until we're closer to merging, but I would like to also include some documentation in the README and the root library documentation about rate limiting.